qa-tests-backend/src/test/groovy/CertExpiryTest.groovy

```diff
@@ -6,6 +6,15 @@ import util.Cert
 
 @Category([BAT, COMPATIBILITY])
 class CertExpiryTest extends BaseSpecification {
+//TODO(ROX-13449): Delete the following test case before merging
+    def "Failing test - Delete before merging"() {
+        when:
+        "This test is supposed to fail"
+
+        then:
+        "Check the behavior of the fail-fast flag in Prow"
+        assert False
+    }
```

Comment:
Have you been able to see it failing already? Is this piece of code now ready to be removed?

Reply:
Yes, seen it failing twice. I'll remove it now.